code/game/gamemodes/cult/cult_items.dm

```diff
@@ -116,19 +116,195 @@
+	desc = "A heavily-armored helmet worn by warriors of the Nar-Sien cult. It can withstand hard vacuum."
 	icon_state = "cult_helmet"
 	item_state = "cult_helmet"
 	armor = list(melee = 60, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30)
 
 /obj/item/clothing/suit/space/cult
-	name = "nar-sian bruiser's armor"
+	name = "nar-sien hardened armor"
 	icon_state = "cult_armor"
 	item_state = "cult_armor"
-	desc = "A heavily-armored exosuit worn by warriors of the Nar-Sian cult. It can withstand hard vacuum."
-	w_class = 3
+	desc = "A heavily-armored exosuit worn by warriors of the Nar-Sien cult. It can withstand hard vacuum."
+	w_class = 2
 	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade,/obj/item/weapon/tank/internals/)
 	armor = list(melee = 70, bullet = 50, laser = 30,energy = 15, bomb = 30, bio = 30, rad = 30)
+
+/obj/item/clothing/suit/space/cult/pickup(mob/living/user)
+	..()
+	if(!iscultist(user))
+		user << "<span class='cultlarge'>\"I wouldn't advise that.\"</span>"
+		user << "<span class='warning'>An overwhelming sense of nausea overpowers you!</span>"
+		user.Dizzy(30)
+		user.Weaken(5)
+
+/obj/item/weapon/sharpener/cult
+	name = "eldritch whetstone"
+	desc = "A block, empowered by dark magic. Sharp weapons will be enhanced when used on the stone."
+	used = 0
+	increment = 5
+	max = 40
+	prefix = "nar-sien"
+
+/obj/item/clothing/suit/cultrobes/cult_shield
+	name = "empowered cultist armor"
+	desc = "Empowered garb which creates a powerful shield around the user."
+	icon_state = "cult_armor"
+	item_state = "cult_armor"
+	armor = list(melee = 50, bullet = 40, laser = 50,energy = 30, bomb = 50, bio = 30, rad = 30)
+	var/current_charges = 3
+	var/shield_state = "shield-red"
+	
+/obj/item/clothing/suit/cultrobes/cult_shield/pickup(mob/living/user)
+	..()
+	if(!iscultist(user))
+		user << "<span class='cultlarge'>\"I wouldn't advise that.\"</span>"
+		user << "<span class='warning'>An overwhelming sense of nausea overpowers you!</span>"
+		user.Dizzy(30)
+		user.Weaken(5)
+
+/obj/item/clothing/suit/cultrobes/cult_shield/hit_reaction(mob/living/carbon/human/owner, attack_text, isinhands)
+	if(current_charges > 0)
+		var/datum/effect_system/spark_spread/s = new
+		s.set_up(2, 1, src)
+		s.start()
+		owner.visible_message("<span class='danger'>[owner]'s shields deflect [attack_text] in a shower of sparks!</span>")
+		current_charges--
+		if(current_charges <= 0)
+			owner.visible_message("[owner]'s shield is destroyed!")
+			shield_state = "broken"
+			owner.update_inv_wear_suit()
+		return 1
+	return 0
+
+/obj/item/clothing/suit/cultrobes/cult_shield/worn_overlays(isinhands)
+    . = list()
+    if(!isinhands)
+        . += image(icon = 'icons/effects/effects.dmi', icon_state = "[shield_state]")
+
+/obj/item/clothing/suit/cultrobes/berserker
+	name = "flagellant's robes"
+	desc = "blood-soaked robes infused with dark magic; allowing the user to move at inhuman speeds, but at the cost of increased damage"
```

Comment:
replace the desc with
`Blood-soaked robes infused with dark magic; allows the user to move at inhuman speeds, but at the cost of increased damage.`